examples/pid-json-example-payload.json

```diff
@@ -10,13 +10,20 @@
   },
   "vct": "https://pidprovider.example.org/v1.0/personidentificationdata",
   "vct#integrity": "c5f73e250fe869f24d15118acce286c9bb56b63a443dc85af653cd73f6078b1f",
-  "verification": {
-    "trust_framework": "eidas",
-    "assurance_level": "high",
-    "evidence": {
-      "method": "cie"
+  "verification": [
```

Comment:
In this non-normative example the `assurance_level` and `evidence` sub-values are missing.